pkg/util/metrics/metrics_test.go

```diff
@@ -0,0 +1,52 @@
+package metrics
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestGetResultByError(t *testing.T) {
+	tests := []struct {
+		name  string
+		error error
+		want  string
+	}{
+		{
+			name:  "error is nil",
+			error: nil,
+			want:  "success",
+		},
+		{
+			name:  "error is not nil",
+			error: errors.New("hello,error"),
+			want:  "error",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetResultByError(tt.error); got != tt.want {
+				t.Errorf("GetResultByError() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestDurationInSeconds(t *testing.T) {
+	tests := []struct {
+		name  string
+		start time.Time
+	}{
+		{
+			name:  "testSeconds",
+			start: time.Date(2022, 10, 1, 7, 8, 9, 0, time.Local),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := DurationInSeconds(tt.start)
+			fmt.Printf("DurationInSeconds() = %v\n", got)
+		})
+	}
+}
```

Comment:
Hi @my-git9, This test does not judge the test result as expected and can be removed.

Reply:
Updated